packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/defaultmodules/uuid/Uuidv5.kt

```diff
@@ -0,0 +1,28 @@
+package expo.modules.kotlin.defaultmodules.uuid
```

Comment:
I guess the 'defaultmodules' folder was intended to be just for modules. What's the best place for such helpers @lukmccall ?

Reply:
I think we can move it to `expo/modules/kotlin/uuid5/*` instead.

Reply:
* `expo/modules/kotlin/uuidv5/*`